ql/src/java/org/apache/hadoop/hive/ql/parse/repl/metric/event/Stage.java

```diff
@@ -129,12 +127,7 @@ public String getReplStats() {
   }
 
   public void setReplStats(String replStats) {
-    // Check the stat string doesn't surpass the RM_PROGRESS column length.
-    if (replStats.length() >= RM_PROGRESS_LENGTH - 2000) {
```

Comment:
why was it 2k lesser than RM_PROGRESS_LENGTH

Reply:
2k chars are consumed by variables other than replStats.